HPCC-17827 DOCS: Update interface changes

g-pan opened the pull request:
Fix HPCC-17827 DOCS: Update interface changes
Document any new interface changes (in this case)
in ECL Watch.

@JamesDeFabia please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-17827 DOCS: Update interface changes

Fix HPCC-17827 DOCS: Update interface changes
Document any new interface changes (in this case)
in ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml:
Stops the scheduled workunit from running.

JamesDeFabia commented on docs/ECLWatch/ECLWa_mods/ECLWatchSrc.xml:
Reschedules the previously descheduled workunit.

JamesDeFabia reviewed (changes requested):
Comments inline

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 60e34bbe1af99b7453f5835eaddcd0f4629c183e
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.558 seconds |


Install hpccsystems-platform-community_6.4.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |95 |95 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK

g-pan committed:
HPCC-17827 DOCS: Update interface changes

Fix HPCC-17827 DOCS: Update interface changes
Document any new interface changes (in this case)
in ECL Watch.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

JamesDeFabia reviewed (approved):
Looks good

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 9e951ffce7a782881b1be3b6ee81aa5e5e46e3c8
Build: success
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.368 seconds |


Install hpccsystems-platform-community_6.4.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |95 |95 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |733 |733 |0 |
|test (thor) |627 |627 |0 |
|test (roxie) |760 |760 |0 |


HPCC Stop: OK
HPCC Uninstall: OK
